ui/js/langs.js

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
I was thinking of adding it there, but placed it in its own, thanks for great idea, i´ll implent that later today